pkg/controller/gittrackobject/watch.go

```diff
@@ -33,16 +32,15 @@ func (r *ReconcileGitTrackObject) watch(obj unstructured.Unstructured) error {
 	}
 
 	// Create new informer
-	log.Printf("Creating new informer for kind %s", obj.GetObjectKind().GroupVersionKind().Kind)
+	r.log.V(3).Info("Creating informer for child kind")
```

Comment:
Should this not include the actual kind in the message?

Reply:
These messages come out something like:

```
1 watch.go:35] gittrackobject-controller "level"=3 "msg"="Creating informer for child kind" "ChildKind"="StorageClass" "ChildName"="aws-efs" "Namespace"="" 
```
which has the child kind as a field, WDYT?

Reply:
Ah, didn't realise that it was using the "decorated" log method. Carry on.